2018/_data/team.json

```diff
@@ -70,6 +70,18 @@
         "url": "/assets/img/team/sergio-ferraz.jpg",
         "alt": "Sergio Ferraz"
       }
+    },
+    {
+      "name": "Christian Sisti",
+      "position": "Team Leader",
+      "role": "Assistant Organiser Barcelona JUG.",
```

Comment:
In this case we all have our Association Role ( Nacho, Adela, Me ) and the rest "Team Member" 
We would like to maintain the same line.

Reply:
Sure no problem, I just give up as Assistant Organiser of JUG as well for coherence.

Reply:
This is done.